document-store/src/main/java/org/hypertrace/core/documentstore/postgres/utils/PostgresUtils.java

```diff
@@ -352,6 +383,23 @@ public static String parseNonCompositeFilter(
     return filterString.toString();
   }
 
+  private static StringBuilder prepareFilterStringForInOperator(
+      final String parsedExpression, final Iterable<Object> values, final Builder paramsBuilder) {
+    final StringBuilder filterStringBuilder = new StringBuilder();
+    filterStringBuilder.append("(");
+    final String collect =
+        StreamSupport.stream(values.spliterator(), false)
+            .map(
+                val -> {
+                  paramsBuilder.addObjectParam(val);
+                  return parsedExpression + " ?? ?";
```

Comment:
I think, the parsedExpression will already contain `?`

Reply:
Tried with integration tests, `?` seem to be there already.